master Fails testing

### Search first

- [x] I searched and no similar issues were found

### Description

The branch `master` can be built using `mvn compile`, but fails testing using `mvn test`:
```bash
...
[ERROR] Failures:
[ERROR]   UpdateIntegrationTest.check execution parameters are correctly replaced:42 Condition not satisfied:

output == """Output of select * from parameter_value_tests order by id: ...
```

### Steps To Reproduce

```bash
git clone https://github.com/liquibase/liquibase.git
git switch master
cd liquibase
mvn compile
mvn test
```

### Expected/Desired Behavior

Compiles and tests without error messages

### Liquibase Version

master

### Database Vendor & Version

_No response_

### Liquibase Integration

_No response_

### Liquibase Extensions

_No response_

### OS and/or Infrastructure Type/Provider

Windows 11 Pro, JDK 24

### Additional Context

_No response_

### Are you willing to submit a PR?

- [ ] I'm willing to submit a PR (Thank you!)

--- a/.mvn/jvm.config
+++ b/.mvn/jvm.config
@@ -1,3 +1,4 @@
 -Dhttp.keepAlive=false
 -Dmaven.wagon.http.pool=false
 -Dmaven.wagon.httpconnectionManager.ttlSeconds=120
+-Dfile.encoding=UTF-8

--- a/liquibase-standard/src/test/groovy/liquibase/parser/core/formattedsql/FormattedSqlChangeLogParserTest.groovy
+++ b/liquibase-standard/src/test/groovy/liquibase/parser/core/formattedsql/FormattedSqlChangeLogParserTest.groovy
@@ -415,26 +415,29 @@ create table table1 (
 );
 """.trim()
 
-    private static final String INVALID_CHANGELOG_WITH_LEXICAL_ERROR =
-"""-- liquibase formatted sql
-
--- changeset postgres:1
-CREATE TABLE public.PersonsMe (
-    PersonID int,
-    LastName varchar(255),
-    FirstName varchar(255),
-    Address varchar(255),
-    City varchar(255)
-);
-
--- changeset postgres:2
-CREATE TABLE public.Persons (
-    PersonID int�
-    LastName varchar(255),
-    FirstName varchar(255),
-    Address varchar(255),
-    City varchar(255)
-);"""
+    // Construct at runtime to prevent Groovy compiler from stripping the invalid character on Windows/non-UTF-8 locales
+    // Using \u0000 (null byte) instead of \uFFFD since it reliably triggers lexical errors across all platforms
+    private static final String INVALID_CHANGELOG_WITH_LEXICAL_ERROR = [
+        "-- liquibase formatted sql",
+        "",
+        "-- changeset postgres:1",
+        "CREATE TABLE public.PersonsMe (",
+        "    PersonID int,",
+        "    LastName varchar(255),",
+        "    FirstName varchar(255),",
+        "    Address varchar(255),",
+        "    City varchar(255)",
+        ");",
+        "",
+        "-- changeset postgres:2",
+        "CREATE TABLE public.Persons (",
+        "    PersonID int" + "\u0000",
+        "    LastName varchar(255),",
+        "    FirstName varchar(255),",
+        "    Address varchar(255),",
+        "    City varchar(255)",
+        ");"
+    ].join("\n")
 
     private static final String INVALID_CHANGELOG_WITH_DUPLICATE_HEADERS =
 """

